content/en/apps/features/integrations/generic.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Generic"
+weight: 1
+description: >
+   Integrate with any system using RESTful APIs
+keywords: generic
+relatedContent: >
+  apps/features/integrations/dhis2
+  apps/features/integrations/rapidpro
+  apps/features/integrations/openmrs
```

Comment:
This section of the forum would be good to include as a link (though it probably does not fit here): https://forum.communityhealthtoolkit.org/c/development/integrations-interoperability/12

Reply:
Do you have an example of where else in the doc site we do this so I can be consistent? Should we revisit other sections to do the same (mapping to the forum)?

Reply:
Sam has been adding a FAQ section to the tutorials- [example](https://docs.communityhealthtoolkit.org/apps/tutorials/application-settings/#frequently-asked-questions)